src/ucp/wireup/select.c

```diff
@@ -1258,9 +1255,11 @@ ucp_wireup_add_bw_lanes(const ucp_wireup_select_params_t *select_params,
                         ucp_tl_bitmap_t tl_bitmap, ucp_lane_index_t excl_lane,
                         ucp_wireup_select_context_t *select_ctx)
 {
-    ucp_ep_h ep                    = select_params->ep;
-    ucp_context_h context          = ep->worker->context;
-    ucp_wireup_select_info_t sinfo = {0};
+    ucp_ep_h ep                               = select_params->ep;
+    ucp_context_h context                     = ep->worker->context;
+    ucp_wireup_select_info_t sinfo            = {0};
+    unsigned dev_count[UCP_MAX_RESOURCES * 2] = {0};
```

Comment:
IMO it would be better/obvious to have a struct with 2 arrays and not split same array to 2 parts